components/widgets/forest-change/tree-loss/selectors.js

```diff
@@ -136,7 +136,7 @@ const parseSentence = createSelector(
     if (totalLoss === 0) {
       sentence = indicator ? noLossWithIndicator : noLoss;
     }
-    if (tropical && totalLoss > 0) {
+    if (tropical && totalLoss > 0 && endYear !== 2023) {
```

Comment:
here as well, we will need to remove this endYear check when we start working on next year's TCL updates